File: VotingApp_Client_WEB/Models/VotingSessionIngress.js
// Define the VotingSessionIngress class
class VotingSessionIngress {
    constructor() {
      this._objectId = "";
      this._title = "";
      this._creator = "";
      this._questions = [];
      this._results = [];
    }
  
    get ObjectID() {
      return this._objectId;
    }
  
    set ObjectID(value) {
      this._objectId = value;
    }
  
    get Title() {
      return this._title;
    }
  
    set Title(value) {
      this._title = value;
    }
  
    get Creator() {
      return this._creator;
    }
  
    set Creator(value) {
      this._creator = value;
    }
  
    get Questions() {
      return this._questions;
    }
  
    set Questions(value) {
      this._questions = value;
    }
  
    get Results() {
      return this._results;
    }
  
    set Results(value) {
      this._results = value;
    }
  
    toString() {
      return `${this._objectId} ${this._title} ${this._creator} ${this._questions} ${this._results}`;
    }
  }
  

File: VotingApp_Client_WPF/VotingSessionIngress.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace VotingApp_Client_WPF
{
    // container class for session data incoming
    public class VotingSessionIngress
    {
        private string? _objectId;
        private string? _title;
        private string? _creator;
        private List<VotingQuestion?>? _questions;
        private List<VotingPost?>? _results;

        public VotingSessionIngress() { }

        [JsonPropertyName("id")]
        public string? ObjectID { get { return _objectId; } set { _objectId = value; } }
        [JsonPropertyName("title")]
        public string? Title { get { return _title;} set { _title = value; } }
        [JsonPropertyName("creator")]
        public string? Creator { get { return _creator; } set { _creator = value; } }
        [JsonPropertyName("questions")]
        public List<VotingQuestion?>? Questions { get { return _questions; } set { _questions = value; } }
        [JsonPropertyName("results")]
        public List<VotingPost?>? Results { get { return _results; } set { _results = value; } }

        public override string ToString()
        {
            return _objectId+" "+_title+" "+_creator+" "+_questions?.ToString()+" "+_results?.ToString();
        }
    }
}
